template/backend.Procfile

```diff
@@ -0,0 +1,2 @@
+release: yarn db:run:migrations
```

Comment:
Maybe rename to `Procfile.backend` and `Procfile.frontend` so that they list next to each other.